code/src/components/Question.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import InputField from "./InputField";
+import "./Question.css";
+
+const Question = ({ id, question_text, input_type, options, onChange, inputValue }) => {
+  return (
+    <>
+      <h2 className="question-text">{question_text}</h2>
+      <InputField
+        name={id}
+        type={input_type}
+        value={inputValue}
+        options={options}
+        onChange={onChange}
+      />
+    </>
+  );
+};
```

Comment:
Stateless component that could perhaps just live up in Form.js

Reply:
I don't understand why this is or exactly what this should look like, but I wrote a follow-up question on your similar comment regarding the stateless `<SubmittedView />` component :)